images/win/scripts/Tests/run-tests.ps1

```diff
@@ -0,0 +1,6 @@
+################################################################################
+##  File:  run-tests.ps1
+##  Desc:  run all pester tests
+################################################################################
+
+Get-ChildItem  | % {  & $_.FullName } 
```

Comment:
I believe this command won't run Pester tests. It will just run powershell files as usual scripts.
You should run tests like `Invoke-Pester -Path <test_folder>\Validate-*.ps1`